ui/app/common/restclient/services/BuildsDAO.js

```diff
@@ -50,6 +50,8 @@
         }
       });
 
+      PageFactory.decorateNonPaged(resource, '_getLastByConfiguration', 'getLastByConfigurationNonPaged');
```

Comment:
Is the use of "decorateNonPaged" essential here? I need to put a note in the DAOs but these methods should be considered deprecated. They introduce potentially serious performance problems for PNC. If it's a lot of work to change this we can let it go in, once we've removed all usages from the codebase though I'd take a stronger line on it.